examples/newrelic-express-apm/src/sdk-v2/sdk.js

```diff
@@ -1,4 +1,4 @@
-const { ClientBuilder } = require('@commercetools/sdk-client-v2')
+const { ClientBuilder } = require('@commercetools/ts-client')
```

Comment:
I think this file it's related to the previous version of the client, it couldn't be touched

Reply:
Sorry @barbara79 I don't really understand. 🤔 Do you mean the v2 SDK?

Reply:
yes, that's correct the client v2.
If your point is to migrate from v2 to v3, it wouldn't be better to create a separate folder for the client v3